src/Tests/dotnet-new.Tests/Approvals/AllWebProjectsWork.CanShowHelp_Mvc.verified.txt

```diff
@@ -12,6 +12,7 @@ Options:
   --dry-run               Displays a summary of what would happen if the given command line were run if it would result in a template creation.
   --force                 Forces content to be generated even if it would change existing files.
   --no-update-check       Disables checking for the template package updates when instantiating a template.
+  --project <project>     The project that should be used for context evaluation.
```

Comment:
@baronfel do we want to keep common options in template help? (they are artificially added)?

Reply:
These additions look good - the options make sense at the command level (dotnet new --help) and the individual template level (dotnet new console --help).